stonesoup/updater/tests/test_recursive.py

```diff
@@ -699,3 +701,802 @@ def test_recursive_ensemble_errors():
     # Run updater
     with pytest.raises(ValueError):
         _ = updater.update(hypothesis)
+
+
+def test_vsbruf_single_step(measurement_model, prediction, measurement, timestamp):
```

Comment:
Is there a way to combine these tests using pytest.mark.parameterise instead of having many repeated tests with only a different updater?